src/phylo/phyplot.jl

```diff
@@ -0,0 +1,285 @@
+@enum nodesym circle=1 square=2 rect=3
+@enum textpos auto=1 alignright=2 alignleft=3 above=4
+@enum plottype phylogram=1 fan=2
+
+type NodeFormat#{S<:AbstractString}
```

Comment:
I guess with the #'s the interpreter will see NodeFormat as an empty parametric type, and similarly for TipFormat - are these to be uncommented once NodeAnnotations work?